zone/ö

```diff
@@ -0,0 +1,2 @@
+ö  IN  NS  ivan.ns.cloudflare.com
+ö  IN  NS  nelly.ns.cloudflare.com
```

Comment:
Domain names can only be ASCII characters, you need to use [Punycode](https://en.wikipedia.org/wiki/Punycode), so your domain should actually be `xn--nda.o`

Reply:
Thank you Jonah!